docs/platform/howto/project-support-center.rst

```diff
@@ -29,10 +74,70 @@ To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
     * Any error messages
     * Any languages or frameworks you are using
 
-#. Select the affected project, service, and a severity level.
+#. Optional: Upload files such as screenshots, logs, or :ref:`HAR files <create-har-files>`.
+   
+   .. important::
+        Aiven support will never ask you to provide sensitive data such as passwords or personal information. You should remove (not redact) any sensitive data in files that you attach to a support ticket.
 
 #. Click **Create ticket**. 
 
-The new support ticket is sent to our support team for review and you also receive a confirmation email. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+If you are on a paid support tier, your account team is also notified when you submit a ticket. 
+
+You can track the status of your tickets on the **My tickets** page. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+
+
+Add participants to a support ticket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can CC other users in an open support ticket by adding their email addresses to the ticket. 
+
+* CCed emails that belong to Aiven users can also add comments and CC other emails on those tickets.
+* Non-users would only be able to reply to emails to add a comment? Can they CC others via email, adding them to the ticket?
```

Comment:
so I've tested it and there's an unexpected twist :) non-users can reply in their email client, but 
a) their response will not be visible in the portal view to other participants
b) the email will still be send and other participants will see it in their email client
c) the reply will be recorded and visible to our agent as an "internal comment"
d) those non-users will be created in zendesk, but since they are not attributed to any organization they won't be able to see the tickets unless they've created them themselves (which is not possible due to our SSO/user management).
![image](https://github.com/aiven/devportal/assets/1419684/5d7fdd44-9c46-4964-b0f9-c8e4f94948ef)
why on earth zendesk made it so I have no clue, but I don't see how to configure it either :(
to make things easier I'd say we should say in public that only existing aiven users should be cc'ed and we'll update the portal to use the list instead of typing in v2.